src/toil/lib/docker.py

```diff
@@ -114,146 +114,67 @@ def _docker(job,
            RM (2) will stop the container and then forcefully remove it from the system
            using `docker rm -f`. This is the default behavior if defer is set to None.
     """
+
+    # default is 1000, which are standard user permissions
+    # set this to 0 to become root
+    # however all output files will have only root permissions
+    defaultUser = 1000
```

Comment:
This might end up running the container as the "wrong" user. If you have only one user on your system, that user has UID 1000 (on Ubuntu, at least). But that isn't the case if you have multiple users on a system, or if you are running something other than Ubuntu. I have UID 501 on my laptop and 1621 on the CBSE systems, for example.

I'd suggest using os.getuid() to get the UID of the current user, rather than hardcoding 1000 as the default.

Reply:
Yes, good call.  I'll include it in the next commit!